app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,66 @@
+<head>
+  <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
+  <script>
+    $(document).ready(function () {
+
+      var images = [
+        "/assets/list.svg",
+        "/assets/close.svg"
+      ];
+
+      var logos = [
+        "assets/BCnameLogo.png",
+        "assets/BCsingleLogo.png"
+      ];
+
+      var imageIndex = 0;
+      var logoId = document.getElementById("logo");
+
+      $(".navbar-toggler").on("click", function () {
+        if( imageIndex == 0) {
+          logoId.className = 'logo-open';
+        } else {
+          logoId.className = 'logo-closed';
+        }
+        imageIndex = (imageIndex + images.length + 1) % (images.length);
+        $("#logo").attr('src', logos[imageIndex]);
+        $("#image").attr('src', images[imageIndex]);
+      });
+    });
```

Comment:
In this part I have two comments:

1. It is preferable not to use JQuery, you can implement this part with turbolinks:

```suggestion
  document.addEventListener("turbolinks:load", function() {
    let images = ["/assets/list.svg", "/assets/close.svg"];
    let logos = ["assets/BCnameLogo.png", "assets/BCsingleLogo.png"];
    let imageIndex = 0;
    let logoId = document.getElementById("logo");
    document.getElementByClassName("navbar-toggler").on("click", function() {
      if (imageIndex == 0) {
        logoId.className = 'logo-open';
      } else {
        logoId.className = 'logo-closed';
      }
      imageIndex = (imageIndex + images.length + 1) % (images.length);
      document.getElementById("logo").attr('src', logos[imageIndex]);
      document.getElementById("image").attr('src', images[imageIndex]);
    });
  });
```

2. This section should be able to be called as a partial from a separate file, so as not to overload this file.

Reply:
I agree with Yil.